### ISSUE TITLE
Toggle inclusive/exclusive operator motions

### ISSUE THREAD

**Issue by timkendrick** [t9md/atom-vim-mode-plus#313]
Pressing `v` after an operator in Vim [toggles inclusive/exclusive mode for the operator's motion](http://vimdoc.sourceforge.net/htmldoc/motion.html#o_v).

For example, `db` will delete from the character **before** the current cursor position to the start of the word, whereas `dvb` will delete the character **at** the current cursor position to the start of the word.

This also applies to linewise motions – e.g. `dj` will delete the current line **and** the one below it, whereas `dvj` will delete **only** the current line.

Any chance of getting this functionality in vim-mode-plus?

**Comment by t9md** [comment]
Yes, I know.
Thanks for the link.
I rarely use this feature, you use this feature in daily basis in pure Vim?

**Comment by timkendrick** [comment]
Thanks for the fast reply! Yeah, during a typical day's coding I'll probably use the characterwise inclusive/exclusive toggle a couple times (I haven't really tried the linewise one), so for me it's a noticeable departure from pure Vim behavior but more of a mild annoyance than a big deal.

For me these minor drawbacks are far outweighed by the overall excellent vim-mode-plus experience (thanks!!) but just thought it was worth raising if you're aiming for closer compatibility with the standard Vim behavior.

**Comment by t9md** [comment]
Yeah I agree with you. Will check to support this feature soon.

**Comment by timkendrick** [comment]
Great, thanks!

**Comment by t9md** [comment]
TODO rough sketch from vim help

```coffeescript
switch
when @target.isLinewise()
  @target.characterwise = true
  @target.characterwise.exclusive = true
when @target.isCharacterwise()
  @target.inclusive = not @target.inclusive # toggle
```

**Comment by t9md** [comment]
released as v0.48.0. I intentionally skip `force-blockwise` feature.(`d ctrl-v j` to force blockwise).
I don't want make things complicated before I cleanup blockwise codes.

**Comment by timkendrick** [comment]
Amazing, thanks so much for this!
